gateway-service/src/main/java/org/zowe/apiml/gateway/security/service/schema/source/AuthSourceService.java

```diff
@@ -15,6 +15,8 @@
  * Interface represents main methods of service which gets the source of authentication and process it.
  */
 public interface AuthSourceService {
+    // Default expiration time for client certificate 15 min
+    Long X509_DEFAULT_EXPIRATION = 15L * 60 * 1000;
```

Comment:
I am not sure if this is the best place to have x509 related fields, and maybe it could be also configurable

Reply:
this will be fixed in upcoming PR